src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodRecordsAndCompactCtors.java

```diff
@@ -62,7 +62,7 @@ public record MySecondRecord() {
     }
 
     // Record component, compact ctor
-    public record MyThirdRecord(String myString) {
+    public record MyThirdRecord(String myString) { // violation
```

Comment:
new violation was discovered as well

Reply:
Please put violation message in comment, we demand this for all new changes.

Reply:
Remove trailing comment from
`@param properties some value // violation`

Reply:
done